check-before-deployment: add selinux (#19703)
pingcap/docs: check-before-deployment.md

    ```

    To get help information of the `tiup cluster exec` command, run the `tiup cluster exec --help` command.

## Disable SELinux

> Use the [getenforce(8)](https://linux.die.net/man/8/getenforce) utility to check if SELinux is disabled or set to permissive. SELinux in Enforcing mode can cause deployment failures. Consult your operating system's documentation for instructions on disabling SELinux.

Reviewer: ```suggestion
Use the [getenforce(8)](https://linux.die.net/man/8/getenforce) utility to check if SELinux is disabled or set to permissive. SELinux in enforcing mode can cause deployment failures. For instructions on disabling SELinux, refer to your operating system's documentation.
```

    ```

    To get help information of the `tiup cluster exec` command, run the `tiup cluster exec --help` command.

## Disable SELinux

> Check with the [`getenforce(8)`](https://linux.die.net/man/8/getenforce) utility to see if SELinux is disabled or set to permissive. When SELinux is in Enforcing mode it can cause deployment failures. Refer to the documentation of your Operating System to see how to disable SELinux.

Reviewer 2: ```suggestion
Use the [getenforce(8)](https://linux.die.net/man/8/getenforce) utility to check if SELinux is disabled or set to permissive. SELinux in Enforcing mode can cause deployment failures. Consult your operating system's documentation for instructions on disabling SELinux.
```

> 

Reviewer 2: ```suggestion
```
